src/api/axios/Culture/cultureAxios.ts

```diff
@@ -0,0 +1,15 @@
+import { AxiosError } from 'axios';
+
+import instance from '../instance';
+
+export const getCultureData = async () => {
+  try {
+    const response = await instance.get(`/api/cultures`);
+    return response.data;
+  } catch (error) {
+    if (error instanceof AxiosError) {
+      const response = error.response;
+      return response;
+    }
+  }
+};
```

Comment:
예림님의 axios instance  데뷔전이라니 너무 잘 쓰셨어요! api 주소는 하드 코딩하기 보다는 따로 상수로 빼서 쓰는게 오류를 방지할 수 있을거에요!
```suggestion
const API_URL = {
   GET_CULTURE_DATA: `/api/cultures`
}
...
 const response = await instance.get(API_URL.GET_CULTURE_DATA);
 ...
```

Reply:
4주차 과제 코드리뷰를 돌면서 다른분들처럼 따로 빼서 사용해보고 싶다고 생각했는데, 이번 합세 덕분에 실제로 해보게되어서 저도 기뻐요!! 감사합니다. 야호..! 

상수부분 관련해서 궁금한게, 
const API_URL = {
   GET_CULTURE_DATA: `/api/cultures`
}
이 부분을 따로 파일 만들지 않고 같은 파일 내에서 사용해도 상관없나요?? 
constants 파일을 따로 만들어서 가져다 써야하는 부분인지 궁금해서 질문 남깁니다! 감사합니다.

Reply:
해당 부분 반영 완료했습니다!